<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Tesla" duration-ms="24700" started-at="2014-07-16T15:04:22Z" finished-at="2014-07-16T15:04:47Z">
    <groups>
    </groups>
    <test name="otc" duration-ms="24700" started-at="2014-07-16T15:04:22Z" finished-at="2014-07-16T15:04:47Z">
      <class name="scripts.otc.EC_Tesla_RNOrderNum_Creation_001">
        <test-method status="PASS" signature="oneTimeSetUp(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:scripts.otc.EC_Tesla_RNOrderNum_Creation_001@e32802]" name="oneTimeSetUp" is-config="true" duration-ms="11234" started-at="2014-07-16T20:34:22Z" finished-at="2014-07-16T20:34:33Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Tesla]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[EC_Tesla_RNOrderNum_Creation_001=Windows 7||Chrome]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[uat]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Local]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[otc]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[OTC]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- oneTimeSetUp -->
        <test-method status="FAIL" signature="RNOrderNum_Creation()[pri:0, instance:scripts.otc.EC_Tesla_RNOrderNum_Creation_001@e32802]" name="RNOrderNum_Creation" duration-ms="13430" started-at="2014-07-16T20:34:33Z" finished-at="2014-07-16T20:34:47Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[command:null||description:null||Fail]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: command:null||description:null||Fail
	at com.thoughtworks.selenium.SeleneseTestBase.fail(SeleneseTestBase.java:391)
	at com.arsin.ArsinSeleniumAPI.reportStepDtlsWithScreenshot(ArsinSeleniumAPI.java:2273)
	at scripts.otc.EC_Tesla_RNOrderNum_Creation_001.RNOrderNum_Creation(EC_Tesla_RNOrderNum_Creation_001.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.TestNG.privateMain(TestNG.java:1364)
	at org.testng.TestNG.main(TestNG.java:1333)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- RNOrderNum_Creation -->
        <test-method status="FAIL" signature="oneTearDown()[pri:0, instance:scripts.otc.EC_Tesla_RNOrderNum_Creation_001@e32802]" name="oneTearDown" is-config="true" duration-ms="0" started-at="2014-07-16T20:34:47Z" finished-at="2014-07-16T20:34:47Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Fail]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Fail
	at com.thoughtworks.selenium.SeleneseTestBase.fail(SeleneseTestBase.java:391)
	at com.arsin.ArsinFrameWork.resultXMLFileCreation(ArsinFrameWork.java:770)
	at com.arsin.ArsinFrameWork.setSauceLabValues(ArsinFrameWork.java:154)
	at com.arsin.ArsinSeleniumAPI.stopSeleniumRunningLocally(ArsinSeleniumAPI.java:2166)
	at com.arsin.ArsinSeleniumAPI.stopSelenium(ArsinSeleniumAPI.java:2141)
	at scripts.otc.EC_Tesla_RNOrderNum_Creation_001.oneTearDown(EC_Tesla_RNOrderNum_Creation_001.java:94)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:225)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:114)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.TestNG.privateMain(TestNG.java:1364)
	at org.testng.TestNG.main(TestNG.java:1333)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- oneTearDown -->
      </class> <!-- scripts.otc.EC_Tesla_RNOrderNum_Creation_001 -->
    </test> <!-- otc -->
  </suite> <!-- Tesla -->
</testng-results>
